openapi.yaml

```diff
@@ -467,10 +467,44 @@ paths:
         '200':
           description: OK
           content: 
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: pong
+                type: object
+                required:
+                  - version
+                properties:
+                  version:
+                    type: string
+                
+  '/account/kin2/{account_address}/status':
+    summary: Get account status
+    get:
+      parameters: 
+        - in: path
+          name: account_address
+          required: true
+          schema:
+            type: string           
+      responses:
+        '200':
+          description: OK
+          content: 
+            application/json:
+              schema:
+                type: object
+                required:
+                  - status
+                properties:
+                  migration:
+                    description: >
+                      * `pending` - Not ready for migration yet
+                      
+                      * `migrate` - Ready and should be migrated
+                      
+                      * `migrated` - Account was migrated and burned
+                    type: string
+                    enum: [pending, migrate, migrated]
```

Comment:
i would use `wallet` instead of `account` to keep it consistent with the rest of our code/ apis

Reply:
It's nice to have pending and migrated, but from client POV, it's the same > i.e. don't do anything.
Do you anticipate the client having something to do with the extra state?